server/services/third-party-apis/eff-aws-ses.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Helpers for working with the EFF Civic CRM APIs.
+ */
+
+var path = require('path');
+var url = require('url');
+
+// load aws sdk
+var aws = require('aws-sdk');
+
+var emailUserCopyOfMessage = function(params, config, cb) {
+
+  awsConfig = config.get('CREDENTIALS.AWS');
+
+
+  // load aws config
+  aws.config.update(awsConfig);
+
+  // load AWS SES
+  var ses = new aws.SES({apiVersion: '2010-12-01'}); //TODO is this version the best one?
+
+  // send to list
+  var to = [params.contact_params.email];
+
+  // this must relate to a verified SES account
+  var from = 'example@example.com'; //TODO change to EFF email verified with AWS SES
```

Comment:
Sorry, I misunderstood here (this *is* `eff-aws-ses.js`). But yeah, let's put it in the config so we don't have to modify a tracked file. I agree it doesn't exactly fit but we could do `API -> AWS_SES_EMAIL`.